hack/collector-default

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+# -------------------------------------------------------------------------------- #
+# Description                                                                      #
+# -------------------------------------------------------------------------------- #
+# collctor-default will collect host information as package for support-bundle-kit #
+# e.g., hostname, syslog, dmesg etc                                                #
+# -------------------------------------------------------------------------------- #
+
+HOST_PATH=$1
+BUNDLE_DIR=$2
+
+cd ${BUNDLE_DIR}
+
+# -------------------------------------------------------------------------------- #
+# host information                                                                 #
+# -------------------------------------------------------------------------------- #
+mkdir -p hostinfos
+cd hostinfos
+
+echo "hostname : " `cat ${HOST_PATH}/etc/hostname` > hostinfo
+echo "system_info : " `uname -a` >> hostinfo
+
+# Boot Configuration
+cat ${HOST_PATH}/boot/config-$(uname -r) > boot_config
+
+# CPU, Memory and Process usage status
+top -b -n 1 > cpu_memory_process_status
+
+# -------------------------------------------------------------------------------- #
+# collect logs                                                                     #
+# -------------------------------------------------------------------------------- #
+cd ${BUNDLE_DIR}
+mkdir -p logs
+cd logs
+
+dmesg &> dmesg.log
+
+# syslog
+tail -c 10m ${HOST_PATH}/var/log/syslog > syslog.log
+
+#kernel log
+chroot ${HOST_PATH} /usr/bin/journalctl -r -k > kernel.log
```

Comment:
Why need chroot here?

Reply:
I think it is because the support-bundle-kit agent executes support-bundle-collector.sh in the container and mounts the volume on `/host`.  Therefore, use chroot to change the root directory of the container process.

https://github.com/rancher/support-bundle-kit/blob/89e14edd2cca7e92df5d475994a0a053a77d1454/pkg/manager/agent.go#L75-L111

Reply:
Got it. Thank you.